### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by a3d4**
For what it's worth.

### CODE REVIEW

File: scripts/release_ppa.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 ## You can pass a branch name as argument to this script (which, if no argument is given,
 ## will default to "develop").
 ##
-## If the gien branch is "release", the resulting package will be uplaoded to
+## If the gien branch is "release", the resulting package will be uploaded to
```
Comment (cameel) [inline scripts/release_ppa.sh]:
```suggestion
## If the given branch is "release", the resulting package will be uploaded to
```

Comment (a3d4) [reply scripts/release_ppa.sh]: Thanks :)! Done.

File: test/libevmasm/Optimiser.cpp [hunk 2]
```diff
@@ -920,8 +920,8 @@ BOOST_AUTO_TEST_CASE(block_deduplicator_loops)
 		AssemblyItem(PushTag, 2),
 		Instruction::JUMP,
 	};
-	BlockDeduplicator dedup(input);
-	dedup.deduplicate();
+	BlockDeduplicator deduplicator(input);
+	deduplicator.deduplicate();
```
Comment (hrkrshnn) [inline test/libevmasm/Optimiser.cpp:924]: Slightly surprised that this is needed. Does the spell-check check the spelling of variables?

Comment (hrkrshnn) [reply test/libevmasm/Optimiser.cpp:924]: Apparently yes.

**Review by hrkrshnn — APPROVED** [review]
LGTM.

**Pull request merged**